crates/typst/src/layout/pages/mod.rs

```diff
@@ -114,9 +114,9 @@ fn layout_pages<'a>(
     for item in &items {
         match item {
             Item::Run(..) => {
-                let layouted = runs.next().unwrap()?;
-                for layouted in layouted {
-                    let page = finalize(engine, &mut counter, &mut tags, layouted)?;
+                let laidout = runs.next().unwrap()?;
+                for laidout in laidout {
```

Comment:
a different correction in the same family. Since it's code, I'm not using `-`s, I'm not sure about conventions, I half imagine that `laid_out` would be more appropriate, but, again, I've left it here for discussion (and dropping isn't hard).